From 3c81fd5d08adfdd6ca96ad3679fd2a6a01d52f80 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Thu, 07 Nov 2019 06:13:34 +0200
Subject: [PATCH] [SW-0] hl: update device name at specs project

Since now we have goya2_arc we need to extract the goya2 from it as
appears at specs directory hirarchy

Change-Id: Iced86e94e164be3a74d365462e9baee102419ac7
---

diff --git a/Makefile b/Makefile
index f73ff9e..2259398 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,7 @@
 CC = $(CROSS_COMPILE)gcc
 OBJCOPY = $(CROSS_COMPILE)objcopy
-CFLAGS += -I$(dir $(BR2_EXTERNAL))/specs/$(HL_DEV)/asic_reg
+SPECS_DEV := $(shell echo $(HL_DEV) | cut -d'_' -f1)
+CFLAGS += -I$(dir $(BR2_EXTERNAL))/specs/$(SPECS_DEV)/asic_reg
 CFLAGS += -I$(dir $(BR2_EXTERNAL))/embedded/specs
 CFLAGS += -Wall -Werror
 CFLAGS += -g -c
